src/components/CharacterEditor/CharacterEditor.module.css

```diff
@@ -41,9 +38,9 @@
 
 .characterWrapper {
   position: fixed;
-  top: 128px;
-  right: 0;
-  width: 500px;
+  left: 60%;
```

Comment:
Leverages our knowledge that the `ControlColumn` is set to 50% width. Doing so within a centered element like `MaxWidthWrapper` ensures that anything inside the control column cannot exceed the midpoint of this container.

So we can reliably set `left: 50%` without overlap. The extra here acts like "margin".